packages/now-cli/src/util/projects/link.ts

```diff
@@ -123,7 +123,11 @@ export async function getLinkedOrg(
 
     return { status: 'linked', org };
   } catch (error) {
-    if (error.code === 'NOT_AUTHORIZED' || error.code === 'TEAM_DELETED') {
+    if (
+      error.status === 403 ||
+      error.code === 'NOT_AUTHORIZED' ||
+      error.code === 'TEAM_DELETED'
```

Comment:
We have this same check in 10+ places so perhaps we need a helper function such as the one we have for env in:`util/env/known-error`

Reply:
I think that would make it worse.

I've already started changing how some errors are rendered and thrown a while ago, the idea is to throw everything that is 5xx, and render the error message (+link) for all 4xx errors in a project wide way

Reply:
The less custom error handling, the better.

Reply:
> I've already started changing how some errors are rendered and thrown 

Is that in another PR? I'm kinda confused by what you mean since thats not what your PR is doing as far as I can see.

Reply:
@styfle no, whenever new utils are added that make API calls, I'll make sure to build them like this, throw on a 5xx and return error + print on a 4xx.

In this case it's not as easy, since it's an existing util that is used in many places.